internal/ng_apf_library/ng_apf_library.bzl

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def _ng_apf_library_impl(ctx):
+    umds = []
+    factories = []
+
+    for file in ctx.files.srcs:
+        if file.basename.endswith(".umd.js"):
+            umds.append(file)
+        elif file.basename.endswith(".ngfactory.js"):
```

Comment:
don't we need ngsummary.js for some tests also?

Reply:
ah yes, `ngsummary.js` is needed for AOT test.